public-html/admindash.php

```diff
@@ -31,6 +31,9 @@
     $menu->render_header();
     $menu->render_menu();
     ?>
+  <div class="main_content">
+  </div>
+
```

Comment:
empty div? Admin dashboard doesn't have anything?

Reply:
Probaly good for a followup

Reply:
No the page is empty right now, I added  div so that whenever content is added it doesn't mess up the UI. I'll raise an issue for adding content in admin dashboard

Reply:
Yes, just a simple message will do, Something should be shown at this page.